hub/api/dataset.py

```diff
@@ -157,6 +158,8 @@ def create_tensor(
         htype: str = DEFAULT_HTYPE,
         chunk_size: int = None,
         dtype: str = None,
+        sample_compression: str = None,
+        chunk_compression: str = None,
```

Comment:
As we discussed, chunk compression only makes sense for very small samples. The total size of all those samples is going to be very small across the dataset, even without compression so it's not really useful.

Reply:
right. but im not removing this in case we need it